Formula/qdated.rb

```diff
@@ -0,0 +1,42 @@
+class Qdated < Formula
+  desc "Create and verify timestamped email addresses for qmail"
+  homepage "https://web.archive.org/web/http://www.palomine.net/qdated/"
+  url "https://web.archive.org/web/http://www.palomine.net/qdated/qdated-0.53.tar.gz"
+  sha256 "002391c656c705838965cab8231261f2474b26711c03e24cf0ecf2978b3cf934"
+
+  bottle do
+    cellar :any_skip_relocation
+  end
```

Comment:
maybe uneeded?
```suggestion
  bottle :unneeded
```

Reply:
if we need bottle artifact, we should remove this block.

Reply:
Thanks for your feedback!

The bottle is needed as it's a compiled formula. However the formula doesn't link against anything, so `:any_skip_relocation` is supported.

If the block should be removed completely: How should I add `:any_skip_relocation` then?

Reply:
`:any_skip_relocation` is added by the bot if it determines that that's an option

Reply:
✅